tests/director/tests/sensitive_data_strip_test.go

```diff
@@ -26,17 +26,13 @@ func TestSensitiveDataStrip(t *testing.T) {
 
 	t.Log("Creating application template")
 	appTmpInput := fixtures.FixApplicationTemplateWithWebhook("app-template-test")
-	var appTemplateID string
-	defer fixtures.DeleteApplicationTemplate(t, ctx, dexGraphQLClient, tenantId, appTemplateID)
 	appTemplate := fixtures.CreateApplicationTemplateFromInput(t, ctx, dexGraphQLClient, tenantId, appTmpInput)
-	appTemplateID = appTemplate.ID
+	defer fixtures.DeleteApplicationTemplate(t, ctx, dexGraphQLClient, tenantId, appTemplate.ID)
```

Comment:
deferring before we try to create the resource was suggested in the previous PR - it makes sence, since the creation could go wrong, and in case of an error somewhere in the function that creates the resource, we'll never reach the defer statement